dt-assets/scss/_main.scss

```diff
@@ -899,3 +899,14 @@ label.dt-switch:active:after {
     list-style-type: none;
     margin: 0;
 }
+
+
+/**
+ * Reveal
+ */
+ div.reveal {
```

Comment:
This means that any modal can't use the h1, h2 and h3 class differently anywhere on D.T
It would be better to create a class modal-header (like `.section-header`) and apply it where needed.
```
div.reveal {
  .modal-header { ... }
}
```

Reply:
My way means that all of those H1, H2, H3 aree styled the same so that when coding you don't have to add classes, but if you want the headings to be different (ie. not use the styling) then add an over-ride styling.